scripts/submitFoundryTx.ts

```diff
@@ -0,0 +1,12 @@
+import { submit } from './utils/submitTx'
+import transactionJson from '../scripts/foundry/transaction.json';
+
+async function main() {
+  console.log(transactionJson)
```

Comment:
is there a way to override the nonce of the safe to which we're posting the transaction?

Reply:
yes this is the 2nd argument in `submit()`, 0 means use the last nonce